### DISCUSSION TITLE
Daily 02/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#515] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer hicimos el sprint review mostrando todo lo que hacía y no hacía el software. Estoy puliendo algunas cosas que quedaron sueltas por ahí sobre todo en documentación, a las 17 tengo inglés. Hoy a las 19 hacemos reunión para grabar el video.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve revisando el IEEE en la wiki y en el archivo. A la tarde tuve clases de programación y por la noche tuvimos la sprint review.
Hoy voy a subir el archivo  IEEE830. Estuve varias hora tratando de que me tome la base de datos de prueba pero no lo conseguí, así que estuve agregando artículos para que el catálogo no aparezca vacío. También estoy revisando el sitio para ver si hay errores y preparandome para hacer el video ya  que por la tarde nos conectamos para realizarlo.

**Comment by BraianTroncoso** [comment]
Ayer estuve realizando actualización de SO y resolviendo unos bugs de espacio en memoria. Por la noche estuve en el sprint review. Hoy modifiqué el boton para que tenga mas sentido con los stylos del sitio, a demas estoy repasando los temas para hacer el video esta noche.

**Comment by Brian-Almada** [comment]
Ayer estuve en el sprint review del grupo. Hoy revisando la página y preparando todo para hacer el demo para la entrega de este sprint.
